### ISSUE TITLE
Missing symbol write_raid56_with_parity

### ISSUE THREAD

**Issue by seblu** [kdave/btrfs-progs#214]
Hello,

libbtrfs version 5.3 requires an undefined symbol `write_raid56_with_parity` and breaks application using it like snapper[1].
```
$ nm -D /usr/lib/libbtrfs.so.0.1|grep raid
                 U write_raid56_with_parity
```

Looks like commit e2da195a3c624b5e862207637e1164b96ed9ccc5 added `extent_io.c` code to libbtrfs which requires this symbol. 

[1] https://bugs.archlinux.org/task/64216

**Comment by maximbaz** [comment]
@seblu FYI #216 contains a working fix, if you want to patch `core/btrfs-progs`.

**Comment by seblu** [comment]
@maximbaz it's in testing now but would be nice to have upstream feedback before moving it to core.

**Comment by kdave** [comment]
Thanks for the report, fix is on the way, now in branch v5.3.x.

**Comment by kdave** [comment]
Fixed in 5.3.1

**Comment by maximbaz** [comment]
thank you!
